modules/apps/transfer/types/transfer_authz.go

```diff
@@ -0,0 +1,116 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/cosmos/cosmos-sdk/x/authz"
+	host "github.com/cosmos/ibc-go/v6/modules/core/24-host"
+
+	"golang.org/x/exp/slices"
+)
+
+const gasCostPerIteration = uint64(10)
```

Comment:
Surprising that `x/authz` doesn't provide a default const out of the box!

Reply:
[Aha...](https://github.com/cosmos/cosmos-sdk/blob/main/x/authz/keeper/keeper.go#L21-L24) 🤔